config/cashier.php

```diff
@@ -83,4 +83,42 @@
         'description' => 'Welcome to ' . config('app.name'),
     ],
 
+    /**
+     * Updating the payment method requires a customer to go through the hosted Mollie checkout to change a Mandate for
```

Comment:
`to change a Mandate`

change into

`to add a new Mandate`

Reply:
There are a few references in these docblocks that `update_payment_method` settings can be overridden per plan. I don't think that's the case here, right?

Reply:
>There are a few references in these docblocks that update_payment_method settings can be overridden per plan. I don't think that's the case here, right?

This is not the case here.